mods_cocina_mappings/mods_to_cocina_subject.txt

```diff
@@ -87,6 +87,31 @@ Option B
   ]
 }
 
+4b. Multi-term subject with authority but no valueURI
+<subject authority="lcsh">
+  <topic>Prices</topic>
+  <geographic>Great Britain</geographic>
+</subject>
+{
+  "subject": [
+    {
+      "structuredValue": [
+        {
+          "value": "Prices",
+          "type": "topic"
+        },
+        {
+          "value": "Great Britain",
+          "type": "place"
+        }
+      ],
+      "source": {

```

Comment:
Can we add the same in https://github.com/sul-dlss-labs/cocina-descriptive-metadata/blob/master/mods_cocina_mappings/mods_to_cocina_subject.txt#L97-L118 so that it's a consistent mapping?

Reply:
Done.